Services/Interfaces/web_interface/models/configuration.py

```diff
@@ -41,8 +42,7 @@
 from octobot_evaluators.evaluator.abstract_evaluator import AbstractEvaluator
 from octobot_backtesting.api.backtesting import is_backtesting_enabled
 from octobot.community.community_manager import CommunityManager
-from octobot_tentacles_manager.api.configurator import save_tentacles_setup_configuration
-from octobot_tentacles_manager.api.inspector import get_tentacle_resources_path
+from octobot.constants import CONFIG_FILE_SCHEMA
```

Comment:
I don't know what to think about this import. Do we really want to import something directly in OctoBot's package ?

Reply:
We don't really have a choice most tentacle import things from the OctoBot package (constants, tests, etc)

Reply:
Oh ok :+1: